AsaApi/Core/Private/Tools/Requests.cpp

```diff
@@ -234,11 +269,59 @@ namespace API
 		return true;
 	}
 
+	bool Requests::CreatePostRequest(const std::string& url, const std::function<void(bool, std::string, std::unordered_map<std::string, std::string>)>& callback,
+		const std::string& post_data, std::vector<std::string> headers, long connectionTimeout, long receiveTimeout, long sendTimeout)
+	{
+		std::thread([this, url, callback, post_data, headers, connectionTimeout, receiveTimeout, sendTimeout]
```

Comment:
I don't love spawning an entire thread per request, threads are quite heavy and can quickly exhaust memory.

Perhaps we should start to look into a threadpool + queue?